iron-multi-selectable.html

```diff
@@ -77,6 +78,18 @@
         (this.selectedValues != null && this.selectedValues.length);
     },
 
+    _updateAttrForSelected: function() {
+      if (this._shouldUpdateSelection) {
+        if (this.multi) {
+          this.selectedValues = this.selectedItems.map(function(selectedItem) {
+            return this._indexToValue(this.indexOf(selectedItem));        
+          }, this);
+        } else {
+          this.selected = this._indexToValue(this.indexOf(this.selectedItem));        
```

Comment:
I think it would be better to call `Polymer.IronSelectableBehavior._updateAttrForSelected)` here instead of duplicating this line.